src/tools-support/tools/index.markdown

```diff
@@ -0,0 +1,19 @@
+---
+title: Downloads
+layout: reference
+---
+
+Below are sample projects you can review to accomplish atomic tasks as you develop your own applications. While originally created for Web Services clients they are helpful to anyone who wants to learn to more about how to use the SAP Concur APIs. They can be downloaded in order, or can be downloaded independently if you’ve already made calls to our APIs and want examples on specific topics like Extracts and Images.
+
+* [API Training Package](/tools-files/APITrainingPackage.zip)
```

Comment:
The tools and tools-files folders are at the same level, so these links need to be the full path: /tools-support/tools-files/APITrainingPackage.zip, etc.